docs/serving/samples/hello-world/helloworld-go/Dockerfile

```diff
@@ -1,24 +1,25 @@
 # Use the offical Golang image to create a build artifact.
 # This is based on Debian and sets the GOPATH to /go.
 # https://hub.docker.com/_/golang
-FROM golang:1.12 as builder
+FROM golang:1.13 as builder
 
 # Copy local code to the container image.
-WORKDIR /go/src/github.com/knative/docs/helloworld
-COPY . .
+WORKDIR /app
+COPY . ./
 
 # Build the command inside the container.
 # (You may fetch or manage dependencies here,
```

Comment:
Delete this comment?

Reply:
Simplified the comment. The Dockerfile spacing and comments are designed to emphasize a pipeline of steps and provide context on how the container is assembled.